be/src/olap/rowset/segment_v2/segment_iterator.h

```diff
@@ -146,6 +146,8 @@ class SegmentIterator : public RowwiseIterator {
     // remember the rowids we've read for the current row block.
     // could be a local variable of next_batch(), kept here to reuse vector memory
     std::vector<rowid_t> _block_rowids;
+    bool _is_need_vec_eval = false;
+    bool _is_need_short_eval = false;
 
     // fields for vectorization execution
     bool _is_all_column_basic_type;
```

Comment:
If `_is_all_column_basic_type` is not used, the definition can be deleted.

Reply:
👌🏻